python-frontend/src/test/java/org/sonar/python/types/v2/UnresolvedImportTypeTest.java

```diff
@@ -111,6 +111,49 @@ void imported_call() {
     assertThat(xType).isInstanceOf(UnknownType.class);
   }
 
+  @Test
```

Comment:
I'm just curious what happens in case of try/except?
Is it propagated as expected as well? What about when the name gets reassigned on top of the try/except? I think it would be good to have a test showing that.